config/src/main/java/org/hyperledger/besu/config/JsonGenesisConfigOptions.java

```diff
@@ -590,4 +590,17 @@ public List<Long> getForks() {
         .sorted()
         .collect(Collectors.toList());
   }
+
+  @Override
+  public List<Long> getForkTimestamps() {
+    Stream<OptionalLong> forkBlockTimestamps = Stream.of(getShanghaiTime(), getCancunTime());
+    // when adding forks add an entry to ${REPO_ROOT}/config/src/test/resources/all_forks.json
```

Comment:
useful comment 👍 

nit: out of scope of your changes but wondering if/why we can't use the already built list from the Schedules?

Reply:
It's done deliberately to catch any mismatches between what we would have in the config file with mixed case and parsing as lower case. There is a pretty good comment about this in the GenesisConfigFileTest https://github.com/hyperledger/besu/blob/main/config/src/test/java/org/hyperledger/besu/config/GenesisConfigFileTest.java#L534.